src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/DefaultValueInstruction.cs

```diff
@@ -23,10 +24,10 @@ internal DefaultValueInstruction(Type type)
         public override string InstructionName => "DefaultValue";
 
         [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2077:UnrecognizedReflectionPattern",
```

Comment:
Do we still get a warning after this change if the attribute is removed?

Reply:
You should, yes, because RuntimeHelpers.GetUninitializedObject is annotated as requiring Constructors on the Type:

https://github.com/dotnet/runtime/blob/c0662e8129beaf93b8050d39a863cc6d16a0308c/src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs#L145-L152